### PR TITLE
docs: js release notes 1.35

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

**Review by dgozman — APPROVED** [review]

File: docs/src/release-notes-js.md [hunk 1]
```diff
@@ -6,6 +6,59 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.35
+
+### Highlights
+
+* UI mode is now available in VSCode Playwright extension via a new "show trace viewer" button:
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:13]:
```suggestion
* UI mode is now available in VSCode Playwright extension via a new "Show trace viewer" button:
```

File: docs/src/release-notes-js.md [hunk 2]
```diff
@@ -6,6 +6,59 @@ toc_max_heading_level: 2
...
+* UI mode and trace viewer mark network requests handled with [`method: Page.route`] and [`method: BrowserContext.route`] handlers, as well as those issued via the [API testing](./api-testing):
+
+  ![Trace Viewer](https://github.com/microsoft/playwright-internal/assets/746130/0a2d3122-3b7d-434d-9bcb-637dc28ce926)
+
+* New option `maskColor` for methods [`method: Page.screenshot`], [`method: Locator.screenshot`], [`method: PageAssertions.toHaveScreenshot#1`] and [`method: LocatorAssertions.toHaveScreenshot#1`] to change default masking color:
+  ```js
+  await page.goto('https://playwright.dev');
+  await expect(page).toHaveScreenshot({
+    mask: [page.locator('img')],
+    maskColor: '#00FF00', // green
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:26]:
```suggestion
    maskColor: 'green',
```
Maybe like that?

File: docs/src/release-notes-js.md [hunk 3]
```diff
@@ -6,6 +6,59 @@ toc_max_heading_level: 2
...
+  $ npx playwright-core install # the new way to install browsers when using playwright-core
+  ```
+
+  This change **does not** affect `@playwright/test` and `playwright` package users.
+
+### Browser Versions
+
+* Chromium 115.0.5790.13
+* Mozilla Firefox 113.0
+* WebKit 16.4
```
Comment (mxschmitt) [inline docs/src/release-notes-js.md:55]:
```suggestion
* WebKit 16.5
```
In theory we should update it

Comment (aslushnikov) [reply docs/src/release-notes-js.md:55]: It is still 16.4 for me on Safari TP 171

File: docs/src/release-notes-js.md (hunk 2 above)
Comment (aslushnikov) [reply docs/src/release-notes-js.md:26]: I'd keep it as-is so that it's clear it supports CSS colors

**Pull request merged**